src/main/resources/edu/cornell/kfs/ksr/businessobject/datadictionary/SecurityGroup.xml

```diff
@@ -0,0 +1,193 @@
+            <list>
+                <ref bean="SecurityGroup-inquirySectionDefinition"/>
+                <ref bean="SecurityGroup-inquirySectionDefinition-securityGroupTab"/>
+            </list>
+        </property>
+    </bean>
+    
+    <bean id="SecurityGroup-inquirySectionDefinition" parent="SecurityGroup-inquirySectionDefinition-parentBean"/>
+    <bean id="SecurityGroup-inquirySectionDefinition-parentBean" parent="InquirySectionDefinition" abstract="true"
+          p:title="">
+        <property name="inquiryFields">
+            <list>
+                <bean parent="FieldDefinition" p:attributeName="securityGroupId"/>
+                <bean parent="FieldDefinition" p:attributeName="securityGroupName"/>
+                <bean parent="FieldDefinition" p:attributeName="securityGroupDescription"/>
+                <bean parent="FieldDefinition" p:attributeName="active"/>
+            </list>
+        </property>
+    </bean>
+    
+    <bean id="SecurityGroup-inquirySectionDefinition-securityGroupTab"
+          parent="SecurityGroup-inquirySectionDefinition-securityGroupTab-parentBean"/>
+    <bean id="SecurityGroup-inquirySectionDefinition-securityGroupTab-parentBean" abstract="true" parent="InquirySectionDefinition"
+          p:id="securityGroupTabSection" p:title="Security Group Tabs">
+        <property name="inquiryFields">
+            <list>
+                <ref bean="SecurityGroup-inquiryCollectionDefinition-securityGroupTabs"/>
+            </list>
+        </property>
+    </bean>
+
+    <bean id="SecurityGroup-inquiryCollectionDefinition-securityGroupTabs"
+          parent="SecurityGroup-inquiryCollectionDefinition-securityGroupTabs-parentBean"/>
+    <bean id="SecurityGroup-inquiryCollectionDefinition-securityGroupTabs-parentBean" abstract="true"
+          parent="InquiryCollectionDefinition" p:attributeName="securityGroupTabs"
+          p:businessObjectClass="edu.cornell.kfs.ksr.businessobject.SecurityGroupTab">
+        <property name="inquiryFields">
+            <list>
+                <bean parent="FieldDefinition" p:attributeName="tabId"/>
+                <bean parent="FieldDefinition" p:attributeName="tabName"/>
+                <bean parent="FieldDefinition" p:attributeName="tabOrder"/>
+                <bean parent="FieldDefinition" p:attributeName="active"/>
+            </list>
+        </property>
+        <property name="summaryFields">
+            <list>
+                <bean parent="FieldDefinition" p:attributeName="tabName"/>
+            </list>
+        </property>
+    </bean>
+    
+    
+    <!-- Business Object Lookup Definition -->
+
+    <bean id="SecurityGroup-lookupDefinition" parent="SecurityGroup-lookupDefinition-parentBean"/>
+    <bean id="SecurityGroup-lookupDefinition-parentBean" abstract="true"
+          parent="SecurityGroup-lookupDefinition-base-parentBean"/>
+    <bean id="SecurityGroup-lookupDefinition-base-parentBean" abstract="true" parent="LookupDefinition"
```

Comment:
Does this really need two levels of parent beans instead of just one parent bean? I don't think you need the extra bean level unless you intend to have another module override part of the configuration (such as the Access Security module).

Reply:
updated